src/sentry/utils/pubsub.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+import redis
+import logging
+import random
+
+from django.conf import settings
+from threading import Thread
+from six.moves.queue import Queue
+
+
+class PubSub():
+    """
+    Poster for a redis pubsub instance.
+
+    put(channel, data) to send data to a redis pubsub
+    channel. Maintains an internal queue for posting, will discard the
+    value if the queue is full or not immediately available.
+    """
+
+    def __init__(self):
+        self._started = False
+
+    def _start(self):
+        if self._started:
+            return True
+
+        connection = getattr(settings, 'PUBSUB_CONNECTION', None)
+        try:
+            host, port, db = connection
+        except (TypeError, ValueError):
+            return False
+        self.rds = redis.StrictRedis(
+            host=host,
+            port=port,
+            db=db,
+            socket_timeout=0.2,
+            socket_connect_timeout=1,
+        )
+
+        self.q = q = Queue(maxsize=100)
+
+        def worker():
+            while True:
+                (channel, data) = q.get()
+                try:
+                    self.rds.publish(channel, data)
+                except Exception:
+                    logger = logging.getLogger('sentry.errors')
+                    logger.exception('could not submit event to pubsub')
```

Comment:
To be clear, these are going to swarm into Sentry. Might be worth reducing the log level here. If for example, this host were down, that means every event is going to create a Sentry event, and we'll kinda eat our own tail a bit.

Reply:
Oh, you're right. Either way, I think we'd want this lower than INFO level otherwise, we might take down our logging infrastructure. And this isn't super useful for us. Either it's working, or it's not. We're not going to get anything useful out of the logging here in production. :)

cc @JTCunning